languages/python/exercises/concept/comments/.docs/instructions.md

```diff
@@ -0,0 +1,15 @@
+In this exercise you'll be processing how to comment effectively.
+
+Exercise 1:
+
+In first function you have to pass a strings.
+
+There are three string are present in this exercise you have to pass only one string to function.If you pass more than one string it should result in error.
```

Comment:
Rewrite this first sentence, there are grammatical errors. Space after the full stop.